multimedia/welle.io/Portfile

```diff
@@ -75,21 +75,18 @@ if {${subport} eq ${name}} {
                             458-fixFreezeWithRtlSdr.patch
 } else {
     # devel
-    github.setup            AlbrechtL welle.io 4b9ab6d00046927309c84ea4aa31df679ce40c59
+    github.setup            AlbrechtL welle.io c3dc6bec7a3494f669efbc0b0792e32fe73a69ab
     set githash             [string range ${github.version} 0 6]
-    version                 20191029+git${githash}
+    version                 20191113+git${githash}
 
     conflicts               welle.io
 
-    checksums               rmd160  1998043b2a1daacbd97abd661b0e9ac6035e209d \
-                            sha256  42b22c2dd8f71e1359c882314b1f3230d22ca580a86c29c075459860c4c4040b \
-                            size    1630258
+    checksums               rmd160  d769bdef25c7c193dca10463213a2f246de64874 \
+                            sha256  8acd45131a4dc9b0e0adadf1e528c215d59617cd1ca3e74643dcbf022d02181e \
+                            size    1634819
 
     configure.pre_args-append \
         -DGIT_COMMIT_HASH=${githash}
-
-    patch.pre_args          -p1
-    patchfiles-append       458-fixFreezeWithRtlSdr.patch
```

Comment:
Since you are removing the reference to this patchfile, it should be deleted in this PR.